src/build/render/components/markdown/index.js

```diff
@@ -34,14 +34,22 @@ const findHeaders = (mdSrc) => {
   return headers;
 };
 
-const renderMarkdown = (md, metaIndex, plaintext) => {
+//renders full markdown texts intended to contain paragraphs
+function renderMarkdown(ctx, md, plaintext) {
   if (!md) return null;
-
-  const tokenizer = buildTokenizer(metaIndex);
-
+  const tokenizer = buildTokenizer(ctx);
   return plaintext ?
     entities.decode(marked(md, {...plaintextRenderOptions, tokenizer})) :
     marked(md, {...htmlRenderOptions, tokenizer});
-};
+}
+
+//renders markdown fragments as inline content rather than as a paragraph
+function renderMarkdownInline(ctx, md, plaintext) {
```

Comment:
Added this inline markdown renderer for things like the "thanks" reasons at the bottom of pages, so we can support markdown there but don't want to render out paragraph tags.